reproman/support/exceptions.py

```diff
@@ -197,3 +197,22 @@ class SSHAccessDeniedViaAuthKeys(BaseException):
     """
     def __init__(self, user):
         self.msg = "access for user '%s' denied via authorized_keys" % user
+
+
+#
+# Run related exceptions
+#
+class JobError(RuntimeError):
```

Comment:
I think it'd make sense to position this next the `OrchestratorError`.

Reply:
true, will do